rules/0625-perdition_rules.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+  -  Perdition (imap/pop3 proxy) rules
+  -  Author: George Kissandrakis <gkissand@gmail.com>
+-->
+<group name="syslog,perdition">
+  <rule id="100100" level="0">

```

Comment:
Hi, rule IDs greater than 100000 are reserved for custom rules. Please change these IDs to others less than 100000.

Thanks.

Reply:
Hi, 
Should I do the rule id changes?
If so, is there an inventory of rule ids not to use an id that is already in use?

Thank you
George

Reply:
Hi @gkissand,

First of all, thanks for your contribution to the project.

I think the best solution here is to add this group of rules to the syslog dedicated file: https://github.com/wazuh/wazuh-ruleset/blob/3.10/rules/0020-syslog_rules.xml

That way we are not creating a new file for three rules related to syslog. You can continue the rule IDs account for that file. I would use IDs 2961-2963.

Regards.